jsonschema2dj/management/custom_field_validators.py

```diff
@@ -0,0 +1,25 @@
+from symbol import factor
+
+from django.core.exceptions import ValidationError
+from django.core import validators
+
+
+class MultipleOfValidator(validators.BaseValidator):
```

Comment:
this class should probably live with the other custom validator here https://github.com/gecBurton/djangorestframework-jsonschema/blob/master/jsonschema2dj/valdiators.py